smile/src/test/java/com/fasterxml/jackson/dataformat/smile/VersionsTest.java

```diff
@@ -1,14 +1,17 @@
 package com.fasterxml.jackson.dataformat.smile;
 
-import java.io.*;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
```

Comment:
@cowtowncoder One thing tho, IntelliJ doesn't provide per-package wildcard handling 🤔
But with that trade off, we get to order imports Jackson way

<img width="735" alt="image" src="https://github.com/user-attachments/assets/f56dd58f-aa3c-47d1-bc50-29d263ee605d" />

Reply:
<img width="686" alt="image" src="https://github.com/user-attachments/assets/05b742ec-166f-4fec-a257-f37b5a1ed19a" />

Reply:
I think that ideally we'd usually left imports as-is and avoid mechanical changes.

But FWTW, the usual cut-off is 5-or-more, I think. Then again for some packages (`java.io`, `java.util`) could use star-include for lower numbers.

I think that order of imports is more important than exact threshold for start includes, if that helps? (i.e. I'd rather have strict(er) ordering than worry about explicit vs star inclusion, if only one can be restricted).